<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="ja-jp">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">dfd374f7168ffdac70251282896c677b634c5a39</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.Activities.WorkflowApplicationTerminatedException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d76b0dd2858d7bb1cef33df90be308d1995e2514</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">a443850f6df25f2a6a78bab9efc8567a3f8f2bae</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>An exception that is thrown when an operation on a workflow instance is not valid because the instance has been terminated.</source>
          <target state="translated">インスタンスが終了したため、ワークフロー インスタンスに対して操作が無効な場合にスローされる例外。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve">
          <source>Creates a new instance of the <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class.</source>
          <target state="translated">新しいインスタンスを作成、 <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>クラスです。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>Creates a new instance of the <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a text string description of the exception.</source>
          <target state="translated">新しいインスタンスを作成、 <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>クラスの例外のテキスト文字列説明を使用します。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>A description of the application terminated exception.</source>
          <target state="translated">アプリケーションの説明には、例外が終了しました。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>Creates a new instance of the <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with the context and serialization information required to retrieve data about this exception from a serialized stream.</source>
          <target state="translated">新しいインスタンスを作成、 <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>シリアル化されたストリームからこの例外に関するデータを取得するために必要なコンテキストおよびシリアル化情報を持つクラス。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>All the information required to serialize or deserialize the application terminated exception object.</source>
          <target state="translated">アプリケーションを逆シリアル化または逆シリアル化に必要なすべての情報は、例外オブジェクトを終了します。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>The source and destination of a given serialized stream and an additional caller-defined context.</source>
          <target state="translated">ソースと指定したシリアル化されたストリームと追加の呼び出し元が定義のコンテキストのコピー先。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>Creates a new instance of the <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a text string description of the exception and with an exception that is the cause of the application terminated exception.</source>
          <target state="translated">新しいインスタンスを作成、 <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>クラス、例外のテキスト文字列説明と、アプリケーションの原因となった例外は例外を終了します。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>A description of the application terminated exception.</source>
          <target state="translated">アプリケーションの説明には、例外が終了しました。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>An exception that has a causal relationship to the application terminated exception.</source>
          <target state="translated">アプリケーションの因果関係がある例外は、例外を終了します。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>The application terminated exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.</source>
          <target state="translated">アプリケーション終了例外は、内部例外への応答がスローされは、内部例外からの情報を使用して、エラーをより適切に処理します。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source>Creates a new instance of the <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a text string description of the exception and with a unique identifier for it.</source>
          <target state="translated">新しいインスタンスを作成、 <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>クラス、例外のテキスト文字列説明と、一意の識別子。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>A description of the application terminated exception.</source>
          <target state="translated">アプリケーションの説明には、例外が終了しました。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>A globally unique identifier.</source>
          <target state="translated">グローバルに一意の識別子。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>Creates a new instance of the <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a text string description of the exception, a unique identifier for it, and an exception that is the cause of the current exception.</source>
          <target state="translated">新しいインスタンスを作成、 <bpt id="p1">&lt;xref href="System.Activities.WorkflowApplicationTerminatedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>例外、一意の識別子、および現在の例外の原因となった例外のテキスト文字列説明クラスです。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>A description of the application terminated exception.</source>
          <target state="translated">アプリケーションの説明には、例外が終了しました。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>A globally unique identifier.</source>
          <target state="translated">グローバルに一意の識別子。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve">
          <source>An exception that has a causal relationship to the application terminated exception.</source>
          <target state="translated">アプリケーションの因果関係がある例外は、例外を終了します。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve">
          <source>The application terminated exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.</source>
          <target state="translated">アプリケーション終了例外は、内部例外への応答がスローされは、内部例外からの情報を使用して、エラーをより適切に処理します。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>